### PR TITLE
1402 forms field

### PR DESCRIPTION
**Pull request by misspran**
## Description
Form field for search panel. Note, purely UI. Form is not fully functional and needs to some actions to wire them up. 
## Screenshots / Demo Video
Screenshot 2023-06-23 at 4 58 11 PM

Wasn't sure what the Posted Within options should be so I put the following, let me know what needs to be changed. 
### Automated and Unit Tests
- [ ] Added Unit tests

### Manual tests for Reviewer
Click on search button on grants page. Search panel should pop up. 

## Checklist
- [ ] Provided ticket and description
- [ ] Provided screenshots/demo
- [ ] Provided testing information
- [ ] Provided adequate test coverage for all new code
- [ ] Added PR reviewers

### CODE REVIEW

File: packages/client/src/components/Modals/SearchPanel.vue [hunk 1]
```diff
@@ -11,12 +11,85 @@
...
+          </b-form-group>
+          <b-form-group label-for="exclude-input">
+            <template slot="label">Exclude Keywords</template>
+              <b-form-input
+                id="exclude-input"
+                type="text"
+                v-model="formData.excludeInput"
+              ></b-form-input>
+          </b-form-group>
+          <b-form-group label-for="oportunity-number">
```
Comment (joshgarza) [inline packages/client/src/components/Modals/SearchPanel.vue:31]: Typo "oportunity -> opportunity" lines 31, 32, 34, 36, 39, 41, 42, 44, 124, 125

**Review by joshgarza — CHANGES_REQUESTED** [review]
This looks good to me! I just noticed a few typos and left a comment with lines to correct.

File: packages/client/src/components/Modals/SearchPanel.vue (hunk 1 above)
Comment (misspran) [reply packages/client/src/components/Modals/SearchPanel.vue:31]: ah, good catch. thank you!

Comment (misspran) [reply packages/client/src/components/Modals/SearchPanel.vue]: should be fixed.

**Review by joshgarza — APPROVED** [review]

**Pull request merged**